CHANGELOG.md

```diff
@@ -5,6 +5,43 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](https://semver.org/).
 
+## [v0.45.0] - 2024-01-29
```

Comment:
Please add the link for `v0.45.0` at the tail of this file.

Reply:
Addressed